specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2021-11-20-preview/elasticsan.json

```diff
@@ -0,0 +1,2658 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-11-20-preview",
```

Comment:
>"2021-11-20-preview" [](http://example.com/codeflow?start=15&length=20)

If this is a new service onboarding, why choosing an older api-version?

Reply:
@jorgecotillo is this a mandatory change ? While we agree, our release date is closer to Jun (hopefully), so should we pick 2022-06-10 for example ?

Reply:
Thanks, @jorgecotillo , in which case, we would like to keep it as-is for this iteration. We might introduce a couple of other incremental features before the preview and we will add the version in that iteration as well.

Please continue with the review and let us know if there are any other issues.